Makefile

```diff
@@ -16,6 +16,11 @@ dockerize:
 	$(MAKE) -C http-gateway			dockerize
 	$(MAKE) -C topic-controller		dockerize
 
+kubectl-apply:
+	$(MAKE) -C kubernetes-crds		kubectl-apply
+	$(MAKE) -C function-controller	kubectl-apply
```

Comment:
Indentation looks wrong, at least on github.

Reply:
Add `?ts=4` to the github URL and everything falls into place :)

I can change to spaces, but I know people on this project prefer tabs (I won't name names)